context/http.go

```diff
@@ -246,11 +246,7 @@ func (ctx *muxVarsContext) Value(key interface{}) interface{} {
 			return ctx.vars
 		}
 
-		if strings.HasPrefix(keyStr, "vars.") {
-			keyStr = strings.TrimPrefix(keyStr, "vars.")
-		}
-
-		if v, ok := ctx.vars[keyStr]; ok {
+		if v, ok := ctx.vars[strings.TrimPrefix(keyStr, "vars.")]; ok {
```

Comment:
This change is unrelated, and looks to be part of https://github.com/distribution/distribution/commit/3b391d32908f3523b525ac9e5785a24a9f488919;

- https://github.com/distribution/distribution/pull/3769

I suggest reverting this; if we want the other changes, we should backport it as a whole and having this partial change will complicate that (result in merge conflicts)

Reply:
again, it's to fix a goci-lint failure.